update tiup document (non-5.0-feature) (#5540)
pingcap/docs: tiup/tiup-component-cluster-display.md

### -N, --node

> - Specifies the node to check. If this option is not specified, all nodes are checked. The value of this option is a comma-separated list of node IDs. If you are not sure about the ID of a node, you can skip this option in the command to show the IDs and status of all nodes in the output.

Reviewer: ```suggestion
- Specifies the node to display. If this option is not specified, all nodes are displayed. The value of this option is a comma-separated list of node IDs. If you are not sure about the ID of a node, you can skip this option in the command to show the IDs and status of all nodes in the output.
```

> - Specifies the node to query. If not specified, all nodes are queried. The value of this option is a comma-separated list of node IDs. You can get the node IDs from the first column of the cluster status table returned by the [`tiup-component-cluster-display`](/tiup/tiup-component-cluster-display.md) command.

Reviewer 2: The Chinese version is "检查". The mistranslation "query" is confusing. Please search similar mistranslations in this repo and correct them. Thanks!
```suggestion
- Specifies the node to check. If this option is not specified, all nodes are queried. The value of this option is a comma-separated list of node IDs. You can get the node IDs from the first column of the cluster status table returned by the [`tiup-component-cluster-display`](/tiup/tiup-component-cluster-display.md) command.
```

Author: Done in https://github.com/pingcap/docs/pull/5540/commits/dac316857598d93d0925c743745174772a3cb0a5~PTAL

- Data type: `STRINGS`

> - If this option is not specified in the command, all nodes are selected by default.

Reviewer 2: ```suggestion
- If this option is not specified in the command, all nodes are checked by default.
```

> **Note:**
>
> If the `-R, --role` option is specified at the same time, only the service nodes that match both the specifications of `-N, --node` and `-R, --role` are checked.

### -R, --role

> - Specifies the role to check. If it is not specified, all roles are checked. The value of this option is a comma-separated list of node roles. If you are not sure about the role deployed on a node, you can skip this option in the command to show the roles and status of all nodes in the output.

Reviewer: ```suggestion
- Specifies the role to display. If it is not specified, all roles are displayed. The value of this option is a comma-separated list of node roles. If you are not sure about the role deployed on a node, you can skip this option in the command to show the roles and status of all nodes in the output.
```

> - Specifies the role to query. If not specified, all roles are queried. The value of this option is a comma-separated list of node roles. You can get the roles of nodes from the second column of the cluster status table returned by the [`tiup-component-cluster-display`](/tiup/tiup-component-cluster-display.md) command.

Reviewer 2: ```suggestion
- Specifies the role to check. If it is not specified, all roles are checked. The value of this option is a comma-separated list of node roles. You can get the roles of nodes from the second column of the cluster status table returned by the [`tiup-component-cluster-display`](/tiup/tiup-component-cluster-display.md) command.
```

- Data type: `STRINGS`

> - If this option is not specified in the command, all roles are checked by default.

Reviewer: ```suggestion
- If this option is not specified in the command, all roles are displayed by default.
```

> **Note:**
>

> > If the `-N, --node` option is specified at the same time, only the service nodes that match both the specifications of `-N, --node` and `-R, --role` are checked.

Reviewer: ```suggestion
> If the `-N, --node` option is specified at the same time, only the service nodes that match both the specifications of `-N, --node` and `-R, --role` are displayed.
```
